limelight.py

```diff
@@ -0,0 +1,142 @@
+import cv2
+import numpy as np
+import imutils
```

Comment:
Are we using imutils? I thought we were using cv2 instead.

Reply:
It was included because CV ball detection uses imutils in their code.